### DISCUSSION TITLE
Is there a JS library that allows me to generate Chia's address from mnemonic words

### DISCUSSION THREAD

**Discussion by xnzhao196** [Chia-Network/chia-blockchain#3104] [General]
Is there a JS library that allows me to generate Chia's address from mnemonic words？

**Comment by mariano54** [comment]
You would need to find an EIP2333 library and then implement CLVM in javascript and then use a bech32m library.
It's a lot of work, and it doesn't exist yet.

**Reply by xnzhao196** [reply]
Does the CHIA team have any plans to do this? That's important for mobile wallets, right?
